lib/interfaces/src/InverterInterface.cpp

```diff
@@ -0,0 +1,185 @@
+#include <InverterInterface.h>
+
+/**
+ * Getters for the data
+ */
+
+InverterStatus_s InverterInterface::get_status() {
+    _feedback_data.status.new_data = false;
```

Comment:
would new_data ever be set to true? if we're always setting it to false before returning it